File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashTable = {}
        k = 0
        for i in range(len(nums)):
            k =  target - nums[i]
            if k in hashTable :
                return [hashTable[k],i]

            hashTable[nums[i]] = i
        return []

File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
     Hashtable<Integer, Integer> o = new Hashtable<>();

        for(int c : nums)
        {
            if(o.containsKey(c))
                return true;
            o.put(c,c);
        }
        return false;
    }
    
}